roles/move_objects_between_buckets/README.md

```diff
@@ -0,0 +1,112 @@
+move_objects_between_buckets
+==================
+
+A role to move objects from one S3 Bucket to another.
+Objects have two options: all objects can be transferred or specific objects are transferred through key prefix.
+If the source bucket is empty, the user has two options: source bucket is deleted or source bucket is kept as an empty bucket.
+
+Requirements
+------------
+
+AWS User Account with the following permission:
+
+* s3:HeadBucket
+* s3:GetBucketOwnershipControls
+* s3:ListObjectsV2
+* s3:ListBucket
+* s3:DeleteObject
+* s3:HeadObject
+* s3:PutObjectAcl
+* s3:CopyObject
+* s3:GetObjectTagging
+* s3:DeleteBucket
+
+Role Variables
+--------------
+
+* **source_bucket**: The name of the Amazon S3 bucket that will have its objects retrieved and then emptied. **Required**
```

Comment:
@taehopark32 can we rename role variables to have role_name as prefix?
so it would be `move_objects_between_buckets_source_bucket` instead of `source_bucket` and so on.

this is needed by `ansible-lint` to pass linters.